Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM alpine:edge
+
+LABEL maintainer="Andreas Peters <support@aventer.biz>"
+
+COPY ./ /home/node
+
+RUN echo http://nl.alpinelinux.org/alpine/edge/testing >> /etc/apk/repositories && \
+    apk add --no-cache git nodejs npm tini websockify && \
+    adduser -D -g 1001 -u 1001 -h /home/node node && \
+    mkdir -p /home/node && \
+    mkdir -p /home/node/.npm-global && \
+    mkdir -p /home/node/app  && \
+    chown -R node: /home/node 
+
+USER node
+
+ENV PATH=/home/node/.npm-global/bin:$PATH
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+
+RUN cd /home/node && \
+    npm install && \
+    npm run build 
+
+USER root
+
+RUN apk del gcc git make g++
+
+USER node
+
+EXPOSE 8080
+ENV MUMBLE_SERVER=mumble.aventer.biz:64738
```

Comment:
Is it standard practice in docker-land to hard-code example values and have the user of the dockerfile replace them manually or is there some way to force the user to provide this piece of configuration before being able to run it?

Reply:
Yes it is standard practice to defined example values with that the container will work. If u want to use the container with your server, u just tell it docker engine during the start command:
 
```
docker run -e MUMBLE_SERVER=myownserver.com:12345 mumble-web
```
So u see, its not really "hardcoded" and you don't have to change the dockerfile. :-)

Reply:
Alright, thanks for the PR.

Reply:
your welcome and thanks for the merge.